package.json

```diff
@@ -104,6 +112,14 @@
       "\\.css$": "identity-obj-proxy"
     }
   },
+  "files": [
+    "dist/*.{js,ts,css}",
+    "dist/components/*.ts",
+    "LICENSE",
+    "ACKNOWLEDGMENTS",
+    "README.md",
+    "package.json"
```

Comment:
BTW, according to the NPM's docs we probably could get rid of  `LICENSE`, `README.md`, and `package.json`. Could you check?
https://docs.npmjs.com/files/package.json

![image](https://user-images.githubusercontent.com/206567/96845170-1280ae00-1459-11eb-8a07-752ab503343f.png)

Reply:
We could. I mentioned this back on #59 too. I'd argue keeping them is a bit more explicit and clear, so it doesn't require any knowledge to know that they are always added. I can cut them out if you'd rather they weren't included though.

Reply:
I think you're right. Let keep them.